internal/olm/operator/bundle/install.go

```diff
@@ -78,6 +78,12 @@ func (i *Install) setup(ctx context.Context) error {
 	i.OperatorInstaller.PackageName = labels["operators.operatorframework.io.bundle.package.v1"]
```

Comment:
should not `operators.operatorframework.io.bundle.package.v1` be a const?

Reply:
Switched to using the labels defined in operator-registry. Good catch.